math/APS.cpp

```diff
@@ -0,0 +1,32 @@
+#include<iostream>
+#include<bits/stdc++.h>
+#include<cmath>
+using namespace std;
+
+int main()
+{
+    int sieve[10000000]={0};
+    for(int i=2;i<10000000;i++)
+    {
+        if(sieve[i]==0)
+        for(int j=1;j*i<10000000;j++)
```

Comment:
Can you use a dynamic structure? For example if the input is ```N``` apply sieve erathostenes of a array/list with only ```N``` elements.